ingredient_phrase_tagger/training/utils.py

```diff
@@ -96,41 +99,11 @@ def getFeatures(token, index, tokens):
 
 def singularize(word):
     """
-    A poor replacement for the pattern.en singularize function, but ok for now.
+    use nltk/Snowballstemmer to singularize word.
```

Comment:
a stemmer will do a lot more than singularize words..  ex "larger" becomes "large"

Reply:
Wouldn't it be better to use nltk.WordNetLemmatizer?